src/content/api/cli.md

```diff
@@ -37,18 +37,19 @@ Read the [installation guide](/guides/installation) if you don't already have we
 ## Commands
 
 webpack-cli offers a variety of commands to make working with webpack easy. By default webpack ships with
-
-| Command   | Alias | Description                                                                                 |
-| --------- | ----- | ------------------------------------------------------------------------------------------- |
-| `init`    | c     | Initialize a new webpack configuration                                                      |
-| `migrate` | m     | Migrate a configuration to a new version                                                    |
-| `loader`  | l     | Scaffold a loader repository                                                                |
-| `plugin`  | p     | Scaffold a plugin repository                                                                |
-| `info`    | i     | Output information about your system and dependencies                                      |
-| `serve`   | s     | Run the webpack Dev Server                                                                  |
-| `bundle`  | b     | Run webpack (default command, can be omitted)                                               |
-| `help`    | h     | Display help for commands and options                                                       |
-| `version` | v     | Output the version of `webpack`, `webpack-cli` and `webpack-dev-server` |
+ 
+| Command      | Usage                                     | Description                                                                     |
+| ------------ | ----------------------------------------- | ------------------------------------------------------------------------------- |
+| `build`      | build|bundle|b [entries...] [options]     | Run webpack (default command, can be omitted).                                  |
+| `configtest` | configtest|t [config-path]                | Validate a webpack configuration.                                               |
+| `help`       | help|h [command] [option]                 | Display help for commands and options.                                          |
+| `info`       | info|i [options]                          | Outputs information about your system.                                          |
+| `init`       | init|c [scaffold...] [options]            | Initialize a new webpack configuration.                                         |
+| `loader`     | loader|l [output-path]                    | Scaffold a loader.                                                              |
+| `plugin`     | plugin|p [output-path]                    | Scaffold a plugin.                                                              |
+| `serve`      | serve|s [options]                         | Run the `webpack-dev-server`.                                                   |
+| `version`    | version|v [commands...]                   | Output the version number of `webpack`, `webpack-cli` and `webpack-dev-server`. |
```

Comment:
```suggestion
| `version`    | version\|v [commands...]                   | Output the version number of `webpack`, `webpack-cli` and `webpack-dev-server`. |
```
You are using `|` inside a table cell and it needs to be escaped with a `\` as per https://github.github.com/gfm/#tables-extension-

Reply:
Ah yes 👍🏻